README.md

```diff
@@ -61,6 +61,18 @@ across the V8 and Test262 projects.
   $ gits push origin migration-twenty-three
   ```
 
+## Runtimes
+
+- [io.js](https://iojs.org/en/index.html)
+  + `brew install iojs`
```

Comment:
I figured that we'd add other instructions for other projects as well. I just wrote these because they are my relevant instructions.